traffic_ops/traffic_ops_golang/monitoring/monitoring_test.go

```diff
@@ -702,6 +701,10 @@ func setupMockGetMonitoringServers(mock sqlmock.Sqlmock, monitor Monitor, router
 			}
 		}
 	}
+	// Add an interface and ip addresses for the monitor cache
+	interfaceRows = interfaceRows.AddRow("monitorCacheInterface", nil, 1500, false, 2)
+	ipAddressRows = ipAddressRows.AddRow("5.6.7.8", "10.0.0.0", true, 2, "monitorCacheInterface")
```

Comment:
Any case or instance where we would need a negative test? Such as missing ip4v/ipv6/both?